From 5a0598bde3edf9d0a597ad1414722ee8d9cf226f Mon Sep 17 00:00:00 2001
From: Jack Lloyd <jack@randombit.net>
Date: Thu, 15 Mar 2018 17:20:43 -0400
Subject: [PATCH] Fix incorrect return value check

---
 src/python/botan2.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/python/botan2.py b/src/python/botan2.py
index 85bf532352..0b591b039e 100755
--- a/src/python/botan2.py
+++ b/src/python/botan2.py
@@ -44,7 +44,7 @@ def load_botan_dll(expected_version):
             dll = CDLL(dll_name)
             dll.botan_ffi_supports_api.argtypes = [c_uint32]
             dll.botan_ffi_supports_api.restype = c_int
-            if dll.botan_ffi_supports_api(expected_version) == 1:
+            if dll.botan_ffi_supports_api(expected_version) == 0:
                 return dll
         except OSError:
             pass
